app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java

```diff
@@ -125,9 +118,8 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
              * Called when a drawer has settled in a completely closed state.
              */
             public void onDrawerClosed(View view) {
-                //getActionBar().setTitle(mTitle);
-                if (mPreferences.getBoolean(Const.SETTINGS_OPEN_DRAWER_ON_START, Const.SETTINGS_OPEN_DRAWER_ON_START_DEFAULT)) {
-                    mPreferences.edit().putBoolean(Const.SETTINGS_OPEN_DRAWER_ON_START, !Const.SETTINGS_OPEN_DRAWER_ON_START_DEFAULT).apply();
+                if (PrefUtils.isFirstStartDone(GdgNavDrawerActivity.this)) {
```

Comment:
Yes, I think that's the problem. I'll also change the name of the method since it's not very good.